package.json

```diff
@@ -56,7 +56,6 @@
     "@oclif/test": "1.2.4",
     "@types/babel-types": "7.0.6",
     "@types/babel__generator": "7.0.2",
-    "@types/chai": "4.1.7",
     "@types/common-tags": "1.8.0",
     "@types/cosmiconfig": "5.0.3",
     "@types/globby": "8.0.0",
```

Comment:
I can't find a reference to globby anywhere either. Do you know what this is?

Reply:
Used once upon a time? I killed it 😄